scripts/azure-templates-bootstrapper.yml

```diff
@@ -79,9 +110,13 @@ jobs:
       # build
       - ${{ if endsWith(parameters.name, '_windows') }}:
         - powershell: .\scripts\retry-command.ps1 -RetryCount ${{ parameters.retryCount }} { .\bootstrapper.ps1 -t ${{ parameters.target }} -v $env:VERBOSITY -c ${{ coalesce(variables.CONFIGURATION, 'Release') }} ${{ parameters.additionalArgs }} }
+          env:
+            JavaSdkDirectory: $(JAVA_HOME)
```

Comment:
Some versions of Xamarin.Android do not look at the JAVA_HOME envvar, so just make sure we set it. See https://github.com/xamarin/xamarin-android-tools/pull/74